doc/operation/performance_tuning_tips.md

```diff
@@ -100,19 +138,24 @@ Due to the available <i>Channel Selectors</i> do not fit our needs, a custom sel
 Basically, the custom <i>Channel Selector</i> type must be configured, together with the mapping of channels per storage. This mapping is configured in the form of:
 
 * Total number of different storages. E.g. if we have a MySQL storage, a CKAN storage and a HDFS storage then `cygnusagent.sources.mysource.selector.storages = 3`. Please observe this apply to different storages of the same type, e.g. if we have a MySQL storage and two different HDFS storages (i.e. different HDFS endpoints), then `cygnusagent.sources.mysource.selector.storages = 3` as well.
-* Subset of channels associated to each storage. The union of all the subsets must be equal to all the channels configured for the source. E.g. if `cygnusagent.sources.mysource.channels = ch1 ch2 ch3 ch4 ch5 ch6` and if `ch1` is associated to a MySQL storage, `ch2` and `ch3` are associated to a CKAN storage and `ch4`, `ch5` and `ch6` are associated to a HDFS storage then `cygnusagent.sources.mysource.selector.storages.storage1 = ch1`, `cygnusagent.sources.mysource.selector.storages.storage2 = ch2,ch3` and `cygnusagent.sources.mysource.selector.storages.storage3 = ch4,ch5,ch6`. 
+* Subset of channels associated to each storage. The union of all the subsets must be equal to all the channels configured for the source. E.g. if `cygnusagent.sources.mysource.channels = ch1 ch2 ch3 ch4 ch5 ch6` and if `ch1` is associated to a MySQL storage, `ch2` and `ch3` are associated to a CKAN storage and `ch4`, `ch5` and `ch6` are associated to a HDFS storage then `cygnusagent.sources.mysource.selector.storages.storage1 = ch1`, `cygnusagent.sources.mysource.selector.storages.storage2 = ch2,ch3` and `cygnusagent.sources.mysource.selector.storages.storage3 = ch4,ch5,ch6`.
```

Comment:
There were no changes form my side in this section. Maybe a line feed or something like that making Github gets crazy.